src/TSTransformer/util/offset.ts

```diff
@@ -16,6 +16,20 @@ export function offset(expression: luau.Expression, value: number) {
 	if (value === 0) {
 		return expression;
 	}
+	// special case to handle adding and removing the offset
```

Comment:
This comment is a bit misleading; After all, the entire function handles adding or removing an offset. I'd reword it to explicitly mention the example case (`offset - 1 + 1`) it solves.